src/app/components/mint-unmint/components/walkthrough/components/walkthrough-blockchain-tag.tsx

```
@@ -1,18 +1,22 @@
 import { HStack, Image, Text } from '@chakra-ui/react';
 
 const blockchainTagPropertyMap = {
-  ethereum: {
+  evm: {
     logo: '/images/logos/ethereum-logo.svg',
     text: 'ON ETHEREUM',
   },
   bitcoin: {
     logo: '/images/logos/bitcoin-logo.svg',
     text: 'ON BITCOIN',
   },
+  xrpl: {
+    logo: '/images/logos/xrp-logo.svg',
+    text: 'ON XRPL',
+  },
```

if the network types were in a global list, like ALL_NETWORKS, then you can loop through them in various situations, and handle each case, and then if there are any missing, you can throw an error or better yet, Typescript will even through an error. So you can know at compile time that there is somewhere in the code that you forgot to handle the new network type. 

so maybe, if it took networks of type Bitcoin | EVM | XRPL and then you add a new type, typescript would blow up. Does that make sense? I wonder if that would be helpful

---

Added NetworkType enum, which includes EVM, BTC, XRPL. Replaced string type with NetworkType enum arguments.